### ISSUE TITLE
Error: data release None no longer supported by the Marvin API. Update to a later MPL or use Marvin's local file access mode instead

### ISSUE THREAD

**Issue by hplana** [sdss/marvin#819]
**Describe the bug**
I try to use Marvin with the python package.
I enter python3, I import marvin
>>> import marvin
There I have a warning:
[WARNING]: /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/marvin/__init__.py:400: MarvinUserWarning: path /Users/plana/sas/dr17/manga/spectro/analysis/v3_1_1/3.1.0/dapall-v3_1_1-3.1.0.fits cannot be found. Setting dapall to None.
  warnings.warn('path {0} cannot be found. Setting dapall to None.'

I am not sure the warning is important.
I then try the example to retrieve an data cube:
>>>my_cube = marvin.tools.Cube('8485-1901')

After a few seconds, I have errors messages, ending with:
brain.core.exceptions.BrainError: Requests Http Status Error: 409 Client Error: CONFLICT for url: https://magrathea.sdss.org/marvin/api/cubes/8485-1901/
Error: data release None no longer supported by the Marvin API. Update to a later MPL or use Marvin's local file access mode instead.

I am not sure what that's means. I know that Marvin had some trouble last week.
Can you help me.
Thanks a lot.

H. PLana

**Comment by bretthandrews** [comment]
This should fix the problem for the mean time until @havok2063 can release a new version: https://github.com/sdss/marvin/issues/810#issuecomment-2176729494

**Comment by havok2063** [comment]
Yeah apologies for the delay in releasing a new version that fixes this.  Hopefully I can make time for this sometime soon.

**Comment by hplana** [comment]
Hi
The fix mentioned by Brett worked on the macos. Thanks. My worry was,
because of others isdues, I was nt sure if the prb was coming from my
instalation or from the server.
Thanks.
Sincerely
H. Plana
